.circleci/config.yml

```diff
@@ -1101,20 +1090,12 @@ workflows:
       - ember-build-prod:
           requires:
             - frontend-cache
-      - build-static-assets:
-          requires:
-            - ember-build-prod
       - publish-static-assets:
-          filters:
-            branches:
-              only:
-                - main
-                - /release\/\d+\.\d+\.x$/
```

Comment:
filters were never relevant because the `requires` chain resolves to `frontend-cache` which only ran on `main`